### PR TITLE
update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by himanshu211raj**
There is lots of punctuation errors in CONTRIBUTING.md, 
I have tried to solve some of them.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ Thank you for considering and taking the time to contribute! Before contributing
 
 ## 📌 Credits :
 
-This project exists thanks to all the people who contribute.
+This project exists just because of you, thanks to all the people who contribute.
```
Comment (Abhishek-kumar09) [inline README.md:55]: change it to `This project exists, thanks to all the people who contribute.`

**Comment by himanshu211raj** [comment]
@Abhishek-kumar09 done

File: README.md (hunk 1 above)
Comment (himanshu211raj) [reply README.md]: > change it to `This project exists, thanks to all the people who contribute.`

kindly check other issues opened by me as well as other pr opened

**Comment by Abhishek-kumar09** [comment]
Thanks @himanshu211raj

**Pull request merged**